lua/luasnip/extras/filetype_functions.lua

```diff
@@ -0,0 +1,44 @@
+-- Check if treesitter is available
+local ok_parsers, ts_parsers = pcall(require, "nvim-treesitter.parsers")
+if not ok_parsers then
+	ts_parsers = nil
+end
+
+local ok_utils, ts_utils = pcall(require, "nvim-treesitter.ts_utils")
+if not ok_utils then
+	ts_utils = nil
+end
+
+local function from_cursor_pos()
```

Comment:
Small style suggestion, since you use this form for functions below
```suggestion
local from_cursor_pos = function()
```
Or revert the functions below to `local function func_name()` since it looks like that is what's used in other places in LuaSnip `¯\_(ツ)_/¯`

Reply:
Oh, yes, I'd also be for `local function somename` everywhere :+1:

Reply:
Nice catch :+1: ty!